README.md

```diff
@@ -135,7 +135,7 @@ This will setup the local configuration (including a password for you) and store
 
 After the server has fully started, terminate it by pressing Ctrl+C .
 
-Now you should generate your SSL certificates (these secure the communications between your local device and the endpoint worker node running the vscode server.)
```

Comment:
This change isn't right if recall correctly. This script is provided on system as part of the module you would have loaded at this stage of the instructions.

So there is no need to download the script and run it.

If this is missing from the module it should be added.

Reply:
Thanks! I have just reverted the description as I have confirmed that the script is automatically loaded when the vscode-server module is loaded.